instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/GrpcAttributesExtractor.java

```diff
@@ -29,5 +42,14 @@ public void onEnd(
     if (status != null) {
       attributes.put(SemanticAttributes.RPC_GRPC_STATUS_CODE, status.getCode().value());
     }
+
+    if (capturedRequestMetadata != null) {
```

Comment:
it doesn't look like null check is needed here

Reply:
taken care of